### PR TITLE
Extends rem mixin to support multiple value arguments.

### PR DESCRIPTION
**Pull request by niallobrien**
Updated the docs & tests to reflect this addition.

### CODE REVIEW

**Comment by niallobrien** [comment]
Added same functionality to the em mixin and added tests.

**Comment by niallobrien** [comment]
Updated the docs and allowed a base font-size to be passed to the rem mixin if required.

File: docs/units/rem.md [hunk 1]
```diff
@@ -16,29 +22,30 @@ $base--font-size ?= 16px
...
 **Example**
 ```css
 .element
-  font-size rem(32) // 32/16 = 2
+  width rem(16 32) // 16/16 = 1, 32/16 = 2
 
 /* CSS */
 .element {
-  font-size: 2em;
+  width: 1rem 2em;
```
Comment (jackbrewer) [inline docs/units/rem.md]: Can we swap this (plus the rest in this doc) out with `margin` too, as `width` doesn't take multiple values.

File: lib/stylus-mixins/units/em.styl [hunk 2]
```diff
@@ -1,11 +1,23 @@
 //
 // EM
-// ==
+// ===
```
Comment (jackbrewer) [inline lib/stylus-mixins/units/em.styl]: Can you revert this line please? Our 'underlines' normally match the title length.

**Comment by jackbrewer** [comment]
Merging - as discussed, I'll make the changes I mentioned in the comments above before releasing.

**Pull request merged**